dtformats/unified_logging.py

```diff
@@ -298,6 +298,8 @@ class TraceV3File(data_format.BinaryDataFile):
       'unified_logging.yaml')
 
```

Comment:
I don't understand why we use class attributes instead of just adding those in the code.

Reply:
You mean regarding the `_CHUNK_TAG` values ? the main reason is that that makes the code easier to understand. Instead of having to figure out what `0x00006001` represents the code now read `_CHUNK_TAG_FIREHOSE`